inscriptions/ISic000106.xml

```diff
@@ -212,16 +217,40 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#TA #AA">
                 <ab>
-                    <lb n="1" xml:id="AKΙAK"/><gap reason="lost" extent="unknown" unit="character" xml:id="AKΙAU"/><w xml:id="AKΙAe"><unclear xml:id="AKΙAo">sir</unclear><supplied reason="undefined" evidence="previouseditor" xml:id="AKΙAy">e<unclear xml:id="AKΙAΙ">s</unclear></supplied></w> <gap reason="lost" extent="unknown" unit="character" xml:id="AKΙAΤ"/>
-                    <lb n="2" xml:id="AKΙAε"/><gap reason="lost" extent="unknown" unit="character" xml:id="AKΙAο"/><num atLeast="2" xml:id="AKΙBA"><unclear xml:id="AKΙBK">II</unclear></num> <g ref="#interpunct" xml:id="AKΙBU">·</g> <w xml:id="AKΙBe">m<unclear xml:id="AKΙBo">a</unclear></w> <gap reason="lost" atLeast="5" atMost="6" unit="character" xml:id="AKΙBy"/><num atLeast="11" xml:id="AKΙBΙ">XI</num> <gap reason="lost" extent="unknown" unit="character" xml:id="AKΙBΤ"/>
-                    <lb n="3" xml:id="AKΙBε"/><gap reason="lost" extent="unknown" unit="character" xml:id="AKΙBο"/><w xml:id="AKΙCA"><supplied reason="lost" xml:id="AKΙCK">impen</supplied>sa</w> <w xml:id="AKΙCU"><hi rend="ligature" xml:id="AKΙCe">pu</hi>blica</w> <w xml:id="AKΙCo"><unclear xml:id="AKΙCy">lo</unclear></w> <gap reason="lost" extent="unknown" unit="character" xml:id="AKΙCΙ"/>
+                    <lb n="1" xml:id="AKΙAK"/><gap reason="lost" extent="unknown" unit="character" xml:id="AKΙAU"/> <w xml:id="AKΙAe" n="5"><unclear xml:id="AKΙAo">sir</unclear> <supplied reason="undefined" evidence="previouseditor" xml:id="AKΙAy">e<unclear xml:id="AKΙAΙ">s</unclear></supplied></w> <gap reason="lost" extent="unknown" unit="character" xml:id="AKΙAΤ"/>
```

Comment:
we've gained a whitespace here between the unclear and the supplied in the first word of this line (sir es) (I'm guessing the supplied with a different reason might be the cause?)